aws/data_source_aws_wafv2_regex_pattern_set.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/wafv2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func dataSourceAwsWafv2RegexPatternSet() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsWafv2RegexPatternSetRead,
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"regular_expression_list": {
```

Comment:
similar comment made in the resource to consider this attribute as a `Set` with `Elem: &schema.Schema{Type: schema.TypeString}` instead

Reply:
I will change this once the schema change in the resource is confirmed.

Reply:
just noting here the schema should remain as is per resource discussion